rotkehlchen/globaldb/upgrades/v4_v5.py

```diff
@@ -0,0 +1,65 @@
+import json
+import logging
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from rotkehlchen.fval import FVal
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBConnection, DBCursor
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def _create_new_tables(cursor: 'DBCursor') -> None:
+    log.debug('Enter _create_new_tables')
+
+    cursor.execute(
+        """
+        CREATE TABLE IF NOT EXISTS default_rpc_nodes (
+            identifier INTEGER NOT NULL PRIMARY KEY,
+            name TEXT NOT NULL,
+            endpoint TEXT NOT NULL,
+            owned INTEGER NOT NULL CHECK (owned IN (0, 1)),
+            active INTEGER NOT NULL CHECK (active IN (0, 1)),
+            weight TEXT NOT NULL,
+            blockchain TEXT NOT NULL
+        );
+        """,
+    )
+
+    log.debug('Exit _create_new_tables')
+
+
+def _populate_rpc_nodes(cursor: 'DBCursor', root_dir: Path) -> None:
+    log.debug('Enter _populate_rpc_nodes')
+
+    with open(root_dir / 'data' / 'nodes.json') as f:
+        nodes_info = json.loads(f.read())
+
+    nodes_tuples = [
+        (node['name'], node['endpoint'], False, True, str(FVal(node['weight'])), node['blockchain'])  # noqa: E501
+        for node in nodes_info
+    ]
+    log.debug(nodes_tuples)
+    cursor.executemany(
+        'INSERT OR IGNORE INTO default_rpc_nodes(name, endpoint, owned, active, weight, blockchain) '  # noqa: E501
```

Comment:
Isn't ignore no-op anyway? The identifier is the primary key

```suggestion
        'INSERT INTO default_rpc_nodes(name, endpoint, owned, active, weight, blockchain) '  # noqa: E501
```